packages/starknet-snap/src/createAccount.ts

```diff
@@ -51,8 +44,7 @@ export async function createAccount(params: ApiParams, silentMode = false, waitM
       `createAccount:\ncontractAddress = ${contractAddress}\npublicKey = ${publicKey}\naddressIndex = ${addressIndexInUsed}`,
     );
 
-    let failureReason = '';
-    let estimateDeployFee: EstimateFee;
+    const failureReason = '';
```

Comment:
shall we still use this ? Maybe in case the account was already deployed ?

Reply:
it will fail in this call, which is the same behaviour as prev logic

Reply:
we dont need this, yes, we should remove